job_controller/job_controller.go

```diff
@@ -49,6 +49,12 @@ type ControllerInterface interface {
 	// Returns the Group Name(value) in the labels of the job
 	GetGroupNameLabelValue() string
 
+	// Returns the Replica Type(key) in the labels of the job
+	GetReplicaTypeLabelKey() string
+
+	// Returns the Replica Index(value) in the labels of the job
+	GetReplicaIndexLabelKey() string
+
```

Comment:
same for this

Reply:
These two also need to be fixed.

https://github.com/kubeflow/common/blob/b615b1c764233de73d0a6f2d20d1a30bac6d45c6/job_controller/job.go#L206

https://github.com/kubeflow/common/blob/b615b1c764233de73d0a6f2d20d1a30bac6d45c6/job_controller/job.go#L103